e2e/wdio/headless/test.e2e.ts

```diff
@@ -250,11 +250,16 @@ describe('main suite 1', () => {
             it(`should horizontally scroll like the native scrollIntoView when passing ${inputDescription} as argument`, async () => {
                 const searchInput = await $('.searchinput')
                 await searchInput.scrollIntoView(input as any)
-                const wdioX = await browser.execute(() => window.scrollX)
+                let wdioX = await browser.execute(() => window.scrollX)
 
                 await browser.execute((elem, _params) => elem.scrollIntoView(_params), searchInput, input as any)
-                const nativeX = await browser.execute(() => window.scrollX)
+                let nativeX = await browser.execute(() => window.scrollX)
 
+                if (Math.floor(wdioX) > Math.floor(nativeX)){
+                    nativeX = nativeX + 1
+                } else if (Math.floor(wdioX) < Math.floor(nativeX)) {
+                    wdioX = wdioX + 1
+                }
```

Comment:
Can you explain this?

Reply:
deltaX or delataY can be only int not float and we're round it to one side or opposite, whereas web scrollIntoView can work with float that's why there is a difference in a pixel that shoud be calculated for the test

Reply:
I don't understand, why are we adding `+1` here? Can you outline in a concrete example?